build-system/server/server-app.js

```diff
@@ -30,12 +30,12 @@ app.locals.delimiters = '<% %>';
  * HTTPS redirect.
  */
 app.use((req, res, next) => {
-  let host = req.headers.host || req.host;
+  let host = req.headers.host || req.hostname || req.host;
```

Comment:
When running in a Cloud IDE, are they not providing HTTPS endpoints for their previews?

Reply:
In the env I'm using, the host is http, fronted by an https proxy at a different url, so I needed a different way of specifying a url that the app will be accessed by the browser, vs the host that the app is running on.

This particular fix though (adding `req.hostname`) was because `req.host` has been deprecaded and I was seeing errors in the logs, but I just checked and saw that they added it back in as allowed in the upcoming [express5](http://expressjs.com/en/5x/api.html#req.host).